backend/backend.go

```diff
@@ -234,11 +235,31 @@ func Initialize(config *Config) (*Backend, error) {
 	b.Daemons = append(b.Daemons, api)
 
 	// Initialize dashboardd
+	var dashboardTLSConfig *types.TLSOptions
+	if config.TLS != nil {
+		// dashboardd TLS config defaults to the same as the apid
+		dashboardTLSConfig = &types.TLSOptions{
+			CertFile:           config.TLS.GetCertFile(),
+			KeyFile:            config.TLS.GetKeyFile(),
+			TrustedCAFile:      config.TLS.GetTrustedCAFile(),
+			InsecureSkipVerify: config.TLS.GetInsecureSkipVerify(),
+		}
+
+		// override the TLS cert/key if user specified them
+		if config.DashboardTLSCertFile != "" && config.DashboardTLSKeyFile != "" {
+			dashboardTLSConfig.CertFile = config.DashboardTLSCertFile
+			dashboardTLSConfig.KeyFile = config.DashboardTLSKeyFile
+		}
+	} else {
+		// important for this to be nil, since there's
+		// a nil check on dashboardd Start()
+		dashboardTLSConfig = nil
+	}
```

Comment:
I think this `else` branch is not necessary, since the declaration of `dashboardTLSConfig` on line 238 would make its value `nil` already (Go's zero value for pointers). What do you think @gbolo?

Reply:
Hi @ccressent , I would love to remove that, however I believe I didn't have much of a choice due to an error when attempting to start dashabordd. I will revisit it later to see why i explicitly set it to nil.

Reply:
Hi @ccressent , I removed the `nil` and tested the options, which seemed to work just fine. Sorry I can't remember why I was so explicit with that `nil`. Thanks for catching that

Reply:
No worries, that's what reviews are for! :)